search-app/src/cmr/search/data/keywords_to_elastic.clj

```diff
@@ -91,6 +91,19 @@
   (let [regex (process-keyword keyword)]
     {:regexp {field regex}}))
 
+(defn- keywords->regex-filter
+  "Create a filter for keyword searches that checks for a loose match on "
+  [field keywords boost]
+  {:weight boost
+   ;; Should the 'and' below actually be an 'or'? Investigate this as part of CMR-1329
+   :filter {:or (concat
+                 (when-let [keywords (:keywords keywords)]
+                  [{:and (map (partial keyword-regexp-filter field) keywords)}])
+                 (when-let [field-keywords (:field-keywords keywords)]
+                  [{:or (concat
+                         (map (partial keyword-regexp-filter field) field-keywords)
+                         (map (partial keyword-regexp-filter field) field-keywords))}]))}})
+
 (defn- keywords->name-filter
```

Comment:
This function has not been updated. I think it's confusing looking at it in github.